types/react-native-touch-id/react-native-touch-id-tests.ts

```diff
@@ -0,0 +1,5 @@
+import { isSupported, authenticate } from 'react-native-touch-id';
+
+isSupported().then(() => {}).catch(() => {});
+
+authenticate('reason').then(() => {}).catch(() => {});
```

Comment:
Sorry, I don't see what you meant. 

it means like this?

`isSupported().then((isOK: boolean) => {}).catch(() => {});`